docs/dev-storm.md

```diff
@@ -6,6 +6,23 @@ title: Storm Compatibility
 Gearpump provides **binary compatibility** for Apache Storm applications. That is to say, users could easily grab an existing Storm jar and run it 
 on Gearpump. This documentation illustrates Gearpump's comapatibility with Storm.  
 
+Here is a table of Storm versions and features supported on Gearpump.
+
+| Storm 0.9.x | Storm 0.10.x |
+| ----------- | -------------|
+| *basic topology | basic topology | 
+| DRPC | DRPC |
+| multi-lang | multi-lang |
+| storm-kafka | storm-kafka |
+| | storm-hdfs|
+| | storm-hbase |
+| | storm-hive |
+| | storm-jdbc |
+| | storm-redis |
+| | flux |
+
+*basic topology means non-Trident topology that doesn't use special features (DRPC, multi-lang, connectors) of Storm.
```

Comment:
So, how many features are not supported? I think that is more important than supported feature list?